### PR TITLE
Document `cml pr` auto–merge fallback

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
Documentation stub for https://github.com/iterative/cml/pull/932; I expect reviewers (hello, reviewers) 😉 to make some substantial changes if I don't do them before they arrive.

### CODE REVIEW

File: content/docs/ref/pr.md [hunk 1]
```diff
@@ -17,8 +17,7 @@ preventing an infinite chain of runs.
 
 Any [generic option](/doc/ref) in addition to:
 
-- `--auto-merge`: Mark the PR/MR for automatic merging after tests pass
-  (unsupported by Bitbucket).
+- `--auto-merge=<merge|rebase|squash>`: Try to merge the PR/MR after creation.
```
Comment (0x2b3bfa0) [inline content/docs/ref/pr.md:20]:
```suggestion
- `--merge` / `--rebase` / `--squash`: Try to merge, rebase-merge or squash-merge the pull request after creation.
```

File: content/docs/ref/pr.md [hunk 2]
```diff
@@ -17,8 +17,7 @@ preventing an infinite chain of runs.
 
 Any [generic option](/doc/ref) in addition to:
 
-- `--auto-merge`: Mark the PR/MR for automatic merging after tests pass
-  (unsupported by Bitbucket).
+- `--merge` / `--rebase` / `--squash`: Try to merge, rebase-merge or squash-merge the pull request after creation.
```
Comment (casperdcl) [inline content/docs/ref/pr.md:20]:
```suggestion
- `--merge`, `--rebase`, or `--squash`: Try to merge, rebase, or squash-merge the created PR after CI tests pass.
```

File: content/docs/ref/pr.md [hunk 3]
```diff
@@ -45,7 +44,8 @@ The `--auto-merge` option enables
 [auto–merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request)
```
Comment (casperdcl) [inline content/docs/ref/pr.md:44]: the line above this should say "The `--merge`/`--rebase`/`--squash` options [...]"

File: content/docs/ref/pr.md [hunk 4]
```diff
@@ -45,7 +44,8 @@ The `--auto-merge` option enables
...
 (GitHub) or
 [merge when pipeline succeeds](https://docs.gitlab.com/ee/user/project/merge_requests/merge_when_pipeline_succeeds.html)
-(GitLab) to merge the pull/merge request as soon as checks succeed.
+(GitLab) to merge the pull request as soon as checks succeed. If this
+feature can't be enabled, `cml pr` will try to merge the pull request directly.
```
Comment (casperdcl) [inline content/docs/ref/pr.md:48]:
```suggestion
waiting for checks isn't supported, `cml pr` will try to merge the pull request immediately.
```

**Review by casperdcl — CHANGES_REQUESTED** [review]

File: content/docs/ref/pr.md [hunk 5]
```diff
@@ -45,7 +44,8 @@ The `--auto-merge` option enables
 [auto–merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request)
 (GitHub) or
 [merge when pipeline succeeds](https://docs.gitlab.com/ee/user/project/merge_requests/merge_when_pipeline_succeeds.html)
-(GitLab) to merge the pull/merge request as soon as checks succeed.
+(GitLab) to merge the pull request as soon as checks succeed. If this
+waiting for checks isn't supported, `cml pr` will try to merge the pull request immediately.
```
Comment (0x2b3bfa0) [inline content/docs/ref/pr.md:48]:
```suggestion
(GitLab) to merge the pull request as soon as checks succeed. If
waiting for checks isn't supported, `cml pr` will try to merge the pull request immediately.
```

**Review by casperdcl — APPROVED** [review]

**Pull request merged**